File: youlexuan-manager-web/src/main/java/com/offcn/sellergoods/controller/UserController.java
package com.offcn.sellergoods.controller;


import com.alibaba.dubbo.config.annotation.Reference;
import com.offcn.entity.PageResult;
import com.offcn.entity.Result;
import com.offcn.pojo.TbUser;
import com.offcn.usercenter.service.UserService;
import org.springframework.web.bind.annotation.RequestBody;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("/usercenter")
public class UserController {

    @Reference
    UserService userService;

    @RequestMapping("/findPage")
    public PageResult findPage(int pageNum, int pageSize, @RequestBody TbUser tbUser){
        PageResult page = userService.findPage(pageNum, pageSize, tbUser);
        return page;
    }


    @RequestMapping("/delete")
    public Result delete(Long[] ids){
        try {
            userService.delete(ids);
            return new Result(true,"删除成功");
        } catch (Exception e) {
            e.printStackTrace();
            return new Result(false,"删除失败");
        }
    }

}

File: youlexuan-shop-web/src/main/webapp/js/controller/userController.js
app.controller('userController',function ($scope,userService,$controller) {

    $controller('baseController',{$scope:$scope});

    $scope.delete = function(){

        //判断
        if($scope.seleceIds == null || $scope.seleceIds.length<=0){
            alert("请选择后再删除");
            return;
        }

        // $http.get('/usercenter/delete.do?ids='+ $scope.seleceIds).success(function (resp) {
        userService.delete($scope.seleceIds).success(function (resp) {
            if(resp.success){
                $scope.reloadList();
                $scope.seleceIds = [];
            }else{
                alert(resp.message);
            }
        });
    }


    //条件的分页查询
    $scope.search = function(pageNum,pageSize,entity){
        userService.findPage(pageNum,pageSize,entity).success(function (resp) {
            $scope.paginationConf.totalItems =  resp.total;
            $scope.list = resp.rows;
        });
    }

});